httpstan/stan_services.cpp

```diff
@@ -177,6 +205,8 @@ PYBIND11_MODULE(stan_services, m) {
   m.def("constrained_param_names", &constrained_param_names, py::arg("data"),
         "Call the ``constrained_param_names`` method of the model.");
   m.def("dims", &dims, py::arg("data"), "Call the ``get_dims`` method of the model.");
+  m.def("log_prob", &log_prob, py::arg("data"), py::arg("unconstrained_parameters"), py::arg("adjust_transform"),
+        "Call the ``log_prob`` method of the model.");
```

Comment:
Shouldn't we change this docstring?

We're calling `stan::model::log_prob_propto` with the model in this case, right? And the model *does* have a `log_prob` method (which we are not using, I believe).